tests/test_algebra_onnx_operators.py

```diff
@@ -22,7 +22,11 @@
     helper, TensorProto, load_model,
     __version__ as onnx__version__
 )
-from test_utils import dump_data_and_model
+from test_utils import dump_data_and_model, TARGET_OPSET as TO11
+
+
+TARGET_OPSET = None
+TARGET_OPSET_11 = TO11
```

Comment:
Why are we naming it TARGET_OPSET_11? Wouldn't this need to be changed for 12, 13 ...?